rotkehlchen/db/optimismtx.py

```diff
@@ -31,11 +31,11 @@ def add_evm_transactions(
             relevant_address,
         )
 
-        tx_tuples = [(tx.l1_fee, tx.tx_hash) for tx in evm_transactions]
+        tx_tuples = [(tx.l1_fee, tx.chain_id.value, tx.tx_hash, tx.chain_id.value) for tx in evm_transactions]  # noqa: E501
```

Comment:
Properly serialize, and move out of loop

```suggestion
        serialized_chainid = tx.chain_id.serialize_for_db()
        tx_tuples = [(tx.l1_fee, serialized_chainid, tx.tx_hash, serialized_chainid) for tx in evm_transactions]  # noqa: E501
```